lib/galaxy/tool_util/toolbox/views/edam.py

```diff
@@ -36,11 +34,8 @@ class EdamPanelMode(str, Enum):
 
 
 class EdamToolPanelView(ToolPanelView):
-    def __init__(self, edam_ontology_path: Optional[str], mode: EdamPanelMode = EdamPanelMode.merged):
-        edam = load_edam_tree(
-            None if not edam_ontology_path or not os.path.exists(edam_ontology_path) else edam_ontology_path
-        )
-        self.edam = edam
+    def __init__(self, datatypes_registry: Registry, mode: EdamPanelMode = EdamPanelMode.merged):
```

Comment:
You can't import the Registry here, as the datatypes module is not a dependency of the tool_util package. Maybe pass the edam property in directly ?